config.xml

```diff
@@ -59,6 +60,7 @@
         <package name="processdump.vm"/>
         <package name="reg_export.vm"/>
         <package name="regshot.vm"/>
+        <package name="rpcview.vm"/>
```

Comment:
I am going to remove `rpcview.vm` because it seems that is breaking in the daily run, likely because a dependency requirement being incompatible with another package dependency requirement. We can add it until we have fixed that.